packages/x-license-pro/src/generateLicense/generateLicense.ts

```diff
@@ -2,18 +2,29 @@ import { md5 } from '../encoding/md5';
 import { base64Encode } from '../encoding/base64';
 
 const licenseVersion = '1';
+
+export type LicenseType = 'pro' | 'premium';
+
 export interface LicenseDetails {
   orderNumber: string;
   expiryDate: Date;
+  // TODO: to be made required once the store is updated
+  type?: LicenseType;
```

Comment:
The scope does sound better. I went with `type` because I was thinking in terms of bundling all components but if we want to have per component license then `scope` is much better.